### DISCUSSION TITLE
How can i find out a color's dominant wavelength?

### DISCUSSION THREAD

**Discussion by amons000** [colour-science/colour#710] [Q&A] [answered]
since I've got a color's CIE-XYZ coordinate, and I try to find out its dominant wavelength.
can I achieve that by this module?

**Comment by Ron024** [comment, marked as answer]
Yes, please see dominant wavelength.
<https://colour.readthedocs.io/en/latest/generated/colour.dominant_wavelength.html#colour.dominant_wavelength>

**Comment by thomasmansencal** [comment]
Hi @Edwardlin-zlt,

Yes, as @Ron024 said this is performed with the definition he linked. Typical usage is as follows:

``` python
>>> import colour
>>> xy = [0.26415, 0.37770]
>>> xy_n = [0.31270, 0.32900]
>>> colour.dominant_wavelength(xy, xy_n)
(array(504.0),
 array([ 0.00369694,  0.63895775]),
 array([ 0.00369694,  0.63895775]))
```

**Comment by amons000** [comment]
Thanks~
You guys done a wonderful job~

**Comment by thomasmansencal** [comment]
Cheers and you are welcome @Edwardlin-zlt !
